lib/chroot-buildpackages.sh

```diff
@@ -149,8 +181,29 @@ chroot_build_packages()
 
 			local target_dir
 			target_dir="${SRC}/cache/buildpkg/${release}-${arch}-v${CHROOT_CACHE_VERSION}"
+			local t_name="${release}-${arch}-v${CHROOT_CACHE_VERSION}"
+			target_dir="${SRC}/cache/buildpkg/${t_name}"
 			local distcc_bindaddr="127.0.0.2"
 
+			if [ ! -f "${target_dir}.tar.xz" ]; then
+				display_alert "Create a clean Environment archive" "${t_name}.tar.xz" "info"
+				local tmp_dir=$(mktemp -d /tmp/debootstrap-XXXXX)
```

Comment:
Mktemp should stay within the Armbian work directory

`${SRC}/.tmp/`

Reply:
Yes. I agree.